tests/contrib/aiohttp/test_middleware.py

```diff
@@ -76,7 +76,7 @@ async def test_param_handler(app_tracer, aiohttp_client, loop, query_string, tra
     if app[CONFIG_KEY].get("trace_query_string"):
         assert query_string == span.get_tag(http.QUERY_STRING)
     else:
-        assert http.QUERY_STRING not in span.meta
+        assert http.QUERY_STRING not in span.get_tags()
```

Comment:
Can we instead just check using this?

```suggestion
        assert span.get_tag(http.QUERY_STRING) is None
```

Reply:
Discussed with @mabdinur offline and agreed such a change is out of scope of this PR.